x-pack/plugins/apm/server/lib/helpers/spans/get_is_using_service_destination_metrics.ts

```diff
@@ -67,28 +68,51 @@ export async function getIsUsingServiceDestinationMetrics({
 }) {
   const { apmEventClient } = setup;
 
-  const response = await apmEventClient.search(
-    'get_has_service_destination_metrics',
-    {
-      apm: {
-        events: [getProcessorEventForServiceDestinationStatistics(true)],
-      },
-      body: {
-        terminate_after: 1,
-        size: 1,
-        query: {
-          bool: {
-            filter: [
-              ...rangeQuery(start, end),
-              ...kqlQuery(kuery),
-              ...getDocumentTypeFilterForServiceDestinationStatistics(true),
-              ...(useSpanName ? [{ exists: { field: SPAN_NAME } }] : []),
-            ],
+  async function getServiceDestinationMetricsCount(
+    query?: QueryDslQueryContainer
+  ) {
+    const response = await apmEventClient.search(
+      'get_service_destination_metrics_count',
+      {
+        apm: {
+          events: [getProcessorEventForServiceDestinationStatistics(true)],
```

Comment:
This will eventually be dynamic?

Reply:
That's fine with me too. Wanted to call the function for consistency but maybe it's more readable if we simply use ProcessorEvent.metric.

Reply:
Actually, we also use `getDocumentTypeFilterForServiceDestinationStatistics(true)` below to build the term query on `metricset.name`. I think it makes sense to use that function, and it would be a little weird to me if we're using the helper functions for the query, but not for `apm.events`, so I think I'll leave it as-is unless you have strong objections.